packages/_vue3-migration-test/src/main.ts

```diff
@@ -19,8 +19,7 @@ if (VUE_COMPAT_MODE === 2) {
      */
     INSTANCE_LISTENERS: 'suppress-warning',
     RENDER_FUNCTION: false,
-    COMPONENT_V_MODEL: false,
-    WATCH_ARRAY: false
```

Comment:
This warning is OK, we could and should fix it as part of the first phase of the migration.